src/plugins/home/public/application/components/homepage/homepage.tsx

```diff
@@ -0,0 +1,278 @@
+  const {
+    chrome: { logos },
+    getBasePath,
+    injectedMetadata: { getBranding },
+    telemetry,
+    toastNotifications,
+  } = getServices();
+  const { isLoading: isWelcomeLoading, showWelcome, onSkip } = useShowWelcome();
+
+  useEffect(() => {
+    if (!error) {
+      return;
+    }
+
+    toastNotifications.addDanger({
+      title: i18n.translate('home.loadingError.title', {
+        defaultMessage: 'Error loading homepage',
+      }),
+      text: i18n.translate('home.loadingError.description', {
+        defaultMessage:
+          'There was an error loading the homepage. Please refresh the page to try again.',
+      }),
+    });
+
+    // TODO: added a toast, but is there a better way to surface this error?
+    // eslint-disable-next-line no-console
+    console.error(error);
+  }, [toastNotifications, error]);
+
+  if (error) {
+    return (
+      <EuiFlexGroup
+        className="home-homepage-body--fill"
+        direction="column"
+        justifyContent="center"
+        alignItems="center"
+      >
+        <EuiFlexItem grow={false}>
+          <EuiIcon type="alert" color="danger" size="xxl" />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <FormattedMessage
+            id="home.errorText"
+            defaultMessage="There was an error loading the homepage."
+          />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    );
+  }
+
+  if (isHomepageLoading || isWelcomeLoading) {
+    return (
+      <EuiFlexGroup
+        className="home-homepage-body--fill"
+        direction="column"
+        justifyContent="center"
+        alignItems="center"
+      >
+        <EuiFlexItem grow={false}>
+          <EuiLoadingSpinner
+            aria-label={i18n.translate('home.loadingSpinner', {
+              defaultMessage: 'Loading homepage',
+            })}
+            size="xl"
+          />
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <FormattedMessage id="home.loadingText" defaultMessage="Loading..." />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    );
+  }
+
+  if (showWelcome) {
+    return (
+      <Welcome
+        urlBasePath={getBasePath()}
+        onSkip={onSkip}
+        telemetry={telemetry}
+        branding={getBranding()}
+        logos={logos}
+      />
+    );
+  }
+
+  const hero = heroes?.[0];
+
+  return (
+    <>
+      {hero && <HeroSection render={hero.render} />}
+      {sections?.map(({ render, title, description, links }, i) => (
+        <Section key={i} title={title} description={description} links={links} render={render} />
+      ))}
+      <EuiHorizontalRule />
+      <Footer />
+    </>
+  );
+};
+
+export const Homepage = () => {
+  const {
+    application: { getUrlForApp },
+    chrome,
+    injectedMetadata: { getBranding },
+  } = getServices();
+
+  const branding = getBranding();
+  // TODO: is there a better way to check if the title is custom?
+  const title =
+    branding.applicationTitle !== 'OpenSearch Dashboards'
```

Comment:
Surprisingly no. Different parts of the code consider the default title to be varying cases of this. Since this is what every other place is using, it should be safe enough.